File: android/NotifyBridge/app/src/main/java/com/alpware/notifybridge/pairing/PairingResponse.kt
package com.alpware.notifybridge.pairing

import com.google.gson.annotations.SerializedName


/**
 * Response returned by the Mac app after a successful pairing request.
 */

data class PairingResponse(
    @SerializedName("type")
    val type: String,

    @SerializedName("host")
    val host: String,

    @SerializedName("port")
    val port: Int,

    @SerializedName("secret")
    val secret: String,

    @SerializedName("name")
    val name: String
)

File: macos/NotifyBridgeMac/NotifyBridgeMac/NotifyBridgeMac/PairingResponse.swift
//
//  PairingResponse.swift
//  NotifyBridgeMac
//
//  Created by Mahmut Alperen Ünal on 22.05.2026.
//

import Foundation

/// Response returned to Android after a successful pairing request.
struct PairingResponse: Codable {
    let type: String
    let host: String
    let port: Int
    let secret: String
    let name: String
}
